portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,26 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Dylon Tjanaka</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Dylon's Portfolio</h1>
+      <a id="pfp-link" href="images/IMG_1502.png"><img id="pfp" src="images/IMG_1502.png" height="500"></a>
+      <p id="about-me">Welcome to my page! I am a rising second-year studying CS at UCLA. I live in Santa Clara, California with my parents and older brother. My interests include algorithms and machine learning and my hobbies include robotics, cubing, and 3D printing.</p>
```

Comment:
If you notice in the previous of this code, it does not wrap.  You'll want to wrap lines around 80 characters so that all viewers will be able display them without having to scroll.

You should be able to configure your editor to do this for you automatically.

Reply:
Done